apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -3722,6 +3722,21 @@ export const extensions: Extensions = {
                     subHeading:
                     "Enable self-service password recovery for users on the login page."
                 },
+                usernameRecovery: {
+                    form: {
+                        fields: {
+                            enableEmailBasedRecovery: {
+                                label: "Enable email based recovery"
+                            },
+                            enableSMSBasedRecovery: {
+                                label: "Enable SMS based recovery"
+                            }
+                        },
+                        smsProviderWarning: "Ensure that an <1>SMS Provider</1> is configured for the SMS feature to work properly.",
+                        recoveryOptionHeading: " Select Recovery Option"
```

Comment:
Can we remove the redundant space?

```suggestion
                        recoveryOptionHeading: "Select Recovery Option"
```